src/content/1.7/modules/symfonyform/_index.md

```diff
@@ -0,0 +1,264 @@
+              <small class="form-text">{{ 'Have specific needs? Edit particular groups to let them see prices or not.'|trans({}, 'Admin.Shopparameters.Help') }}</small>
+            </div>
+          </div>
+          <div class="form-group row catalog-mode-option">
+            {{ 'Generatable text type'|trans({}, 'Modules.DemoSymfonyForm.Admin') }}
+            <div class="col-sm">
+              {{ form_errors(demoConfigurationTextForm.generatable_text_type) }}
+              {{ form_widget(demoConfigurationTextForm.generatable_text_type) }}
+            </div>
+          </div>
+          {% block product_general_preferences_form_rest %}
+            {{ form_rest(generalForm) }}
+          {% endblock %}
+        </div>
+    </div>
+    <div class="card-footer">
+      <div class="d-flex justify-content-end">
+        <button class="btn btn-primary float-right" id="save-button">
+          {{ 'Save'|trans({}, 'Admin.Actions') }}
+        </button>
+      </div>
+    </div>
+  </div>
+  {{ form_end(demoConfigurationTextForm) }}
+{% endblock %}
+```
+
+In 1.7.8 major updates were made to PrestaShop's form theme in order to make working with forms as straightforward as possible.
+Now it's enough to add template file once and never go back to it again. All further modifications can be done using form type.
+
+```twig
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+
+{# PrestaShop made some modifications to the way forms are displayed to work well with PrestaShop in order to benefit from those you need to use ui kit as theme#}
+{% form_theme demoConfigurationTextForm 'PrestaShopBundle:Admin/TwigTemplateForm:prestashop_ui_kit.html.twig' %}
+
+{% block content %}
+  {{ form_start(demoConfigurationTextForm) }}
+  <div class="card">
+    <h3 class="card-header">
+      <i class="material-icons">settings</i> {{ 'Text form types'|trans({}, 'Modules.DemoSymfonyForm.Admin') }}
+    </h3>
+    <div class="card-block row">
+      <div class="card-text">
+        {{ form_widget(demoConfigurationTextForm) }}
+      </div>
+    </div>
+    <div class="card-footer">
+      <div class="d-flex justify-content-end">
+        <button class="btn btn-primary float-right" id="save-button">
+          {{ 'Save'|trans({}, 'Admin.Actions') }}
+        </button>
+      </div>
+    </div>
+  </div>
+  {{ form_end(demoConfigurationTextForm) }}
+{% endblock %}
+```
+
+
+{{% notice note %}}
+Some form types may require JS components. You can always check documentation for specific form type if it requires one.
+You can find out more about JS components here: https://devdocs.prestashop.com/1.7/development/components/global-components/
+{{% /notice %}}
+
+
+## Form handling
+
+At this point you should have form rendered successfully, so it's time to handle the form.
+
+Usually you handle the form in the same action that renders form tempalate. You just need to check if the form is submitted using ``isSubmitted`` function.
+```php
+if ($textForm->isSubmitted()) {
+}
+``` 
```

Comment:
Before this step, should the documentation describe how to handle the form from the current request ?
```php
$textForm->handleRequest($request);
```